UCLA Radio/UCLA Radio/menu/ShowViewController.swift

```diff
@@ -51,9 +118,20 @@ class ShowViewController: BaseViewController {
         }
     }
     
+    //func getLocalizedTimeString(showTime: DateComponents)
+    
     fileprivate func styleForShow(_ show: Show) {
-        let string = show.day + " " + show.time
-        timeLabel.text = string
+        
+        let formatter = DateFormatter()
+        formatter.amSymbol = formatter.amSymbol.lowercased()
+        formatter.pmSymbol = formatter.pmSymbol.lowercased()
+        
+        // Format: Shorterned day of week (EEE), Shortened 12 hour (h), AM/PM (a)
+        formatter.dateFormat = "EEE ha"
+        
+        let showDate = show.getClosestDateOfShow()
+        
+        timeLabel.text = formatter.string(from: showDate)
```

Comment:
Let's keep concerns separated.. Can we move the time formatting to be a class method in the `String+Dates` extension you made?